its/ruling/src/test/resources/autoscan/autoscan-diff-by-rules.json

```diff
@@ -1334,7 +1334,7 @@
   {
     "ruleKey": "S2629",
     "hasTruePositives": true,
-    "falseNegatives": 27,
+    "falseNegatives": 58,
```

Comment:
After splitting the `LazyArgEvaluationCheck.java` resource in compiling and not-compiling code, and moving it to `java-checks-tests-sources` the compiling resource is analyzed by autoscan and more FN are found!
Namely, all the Noncompliant code is reported; also other rules are hitting on `LazyArgEvaluationCheck.java` resource